StripeConnect/StripeConnect/Source/Internal/Extensions/WKScriptMessage+extension.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  WKScriptMessage+extension.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/22/24.
+//
+
+import WebKit
+
+extension WKScriptMessage {
+    func toDecodable<DecodableType: Decodable>() throws -> DecodableType {
+        if let payload = body as? DecodableType {
+            return payload
+        } else {
```

Comment:
minor nit: The `else` block isn't necessary since `if` returns. It just removes a level of nesting making it a little easier to read